components/dashboards-web-component/src/designer/components/PageCard.jsx

```diff
@@ -151,6 +151,18 @@ class PageCard extends Component {
                                 disabled={page.id === this.props.landingPageId}
                                 onCheck={() => this.props.updateLandingPage(this.props.page.id)}
                             />
+                            {
+                                /* Show hide page option only for non-landing page pages. */
+                                page.id !== this.props.landingPageId &&
+                                <Checkbox
+                                    label={<FormattedMessage id="designer.hide.page" defaultMessage="Hide Page" />}
+                                    defaultChecked={page.hidden}
+                                    onCheck={(e, checked) => {
```

Comment:
Add 
```js
disabled={ page.id === this.props.landingPageId }
```
so the checkbox will be disabled for the landing page.

Reply:
What is the purpose of disabling and showing, when there is no purpose?

Reply:
Consistent UI. We shouldn't surprise the user with sudden UI changes (like appearing a new checkbox).
https://ux.stackexchange.com/a/112250/49172
https://ux.stackexchange.com/a/12761/49172

Page card for all pages should look same. Some pages can be special, like the landing page. In that case, some options might not be available such as `Hide`.